site/features.md

```diff
@@ -3,622 +3,262 @@ title: Weave Features
-connected.
+###<a name="virtual-ethernet-switch"></a>Virtual Ethernet Switch
 
-Containers can easily access services from each other; e.g. in the
-container on `$HOST1` we can start a netcat "service" with
+To application containers, the network established by Weave 
+resembles a giant Ethernet switch, where all containers are 
+connected and can easily access services from one another. 
 
-    root@a1:/# nc -lk -p 4422
+Because Weave uses standard protocols, your favorite network 
+tools and applications, developed over decades, can still 
+be used to configure, secure, monitor, and troubleshoot 
+a container network. 
 
-and then connect to it from the container on `$HOST2` with
+Broadcast and Multicast protocols can also be implemented 
+over Weave Net.
 
-    root@a2:/# echo 'Hello, world.' | nc a1 4422
+To start using Weave Net, see [Installing Weave Net](/site/installing-weave.md)
+and [Deploying Applications to Weave Net](/site/using-weave/deploying-applications.md)
 
-Note that *any* protocol is supported. Doesn't even have to be over
-TCP/IP, e.g. a netcat UDP service would be run with
+###<a name="fast-data-path"></a>Fast Datapath
 
-    root@a1:/# nc -lu -p 5533
+Weave automatically chooses the fastest available method to 
+transport data between peers. The best performing of these 
+(the 'fastdp') offers near-native throughput and latency. 
 
-    root@a2:/# echo 'Hello, world.' | nc -u a1 5533
+Fastdp does not support encryption. If you supply a 
+password at `weave launch` the router falls back to a slower 
+`sleeve` mode that does support encryption. 
 
-Broadcast and multicast protocols also work over Weave Net.
+For connections that traverse untrusted networks, 
+see [Securing Containers Across Untrusted Networks](/site/using-weave/security-untrusted-networks.md) for more details.
 
-We can deploy the entire arsenal of standard network tools and
-applications, developed over decades, to configure, secure, monitor,
-and troubleshoot our container network. To put it another way, we can
-now re-use the same tools and techniques when deploying applications
-as containers as we would have done when deploying them 'on metal' in
-our data centre.
+See [Using Fast Datapath](/site/fastdp/using-fastdp.md) and 
+[How Fast Datapath Works](/site/fastdp/fastdp-how-it-works.md). 
 
-### <a name="fast-data-path"></a>Fast data path
+###<a name="docker"></a>Seamless Docker Integration (Weave Docker API Proxy)
 
-Weave automatically chooses the fastest available method to transport
-data between peers. The most performant of these ('fastdp') offers
-near-native throughput and latency but does not support encryption;
-consequently supplying a password will cause the router to fall back
-to a slower mode ('sleeve') that does, for connections that traverse
-untrusted networks (see the [security](#security) section for more
-details).
+Weave includes a [Docker API Proxy](/site/weave-docker-api/set-up-proxy.md), which can be 
```

Comment:
The emphasis here needs to be not that you specifically use the proxy to launch containers, but that you use the standard Docker command-line client or any other Docker API client, and the proxy silently does its job in the background.